### PR TITLE
Issue #4941 , handle error 'Please specify the Rails release the migr…

### PR DESCRIPTION
**Pull request by oystersauce8**
Issue #4941 , handle error 'Please specify the Rails release the migration was written for'

### CODE REVIEW

File: lib/generators/active_record/devise_generator.rb [hunk 1]
```diff
@@ -82,8 +82,8 @@ def inet?
         postgresql?
       end
 
-      def rails5?
-        Rails.version.start_with? '5'
+      def rails5_and_up?
+        Rails::VERSION::MAJOR >= 5
```
Comment (rafaelfranca) [inline lib/generators/active_record/devise_generator.rb]: This constant is private. Maybe `Rails.gem_version >= Gem::Version('5.0.0')`?

Comment (rafaelfranca) [reply lib/generators/active_record/devise_generator.rb]: Ah, it is public, so to is fine.

**Comment by rafaelfranca** [comment]
Can you also test devise with Rails 6. Right now it is not tested so I don't want to recommend people to use it.

**Comment by oystersauce8** [comment]
My [new app](https://github.com/oystersauce8/untitled) worked ok I guess, basic [tests](https://github.com/oystersauce8/untitled/tree/master/test/integration) passed and running on heroku at https://untitled-d68f47d.herokuapp.com/

The only caveat was the responders gem still didn't "recognize" rails 6 - I sent a [PR](https://github.com/oystersauce8/responders/pull/1/files)

**Pull request merged**